src/aiida/orm/nodes/data/array/bands.py

```diff
@@ -748,8 +748,8 @@ def _matplotlib_get_dict(
         if labels:
             tick_pos, tick_labels = zip(*labels)
         else:
-            tick_pos = []
-            tick_labels = []
+            tick_pos = ()
+            tick_labels = ()
```

Comment:
```suggestion
tick_pos, tick_labels = zip(*labels) if labels else [], []
```

Makes both `tuple | list`. Not sure if this is okay downstream.

Reply:
I think it is 🙂